Cart/Mappers/Mappers.ProductItem.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Cart.Business.Models;
+using Cart.Models;
+
+namespace Cart.Mappers
+{
+    public static partial class Mappers
+    {
+        public static ProductItemViewModel ToView(this ProductItemEntity entity)
+            => new()
+            {
+                Id = entity.Id,
```

Comment:
Use [Automapper](https://docs.automapper.org/en/stable/Getting-started.html) nuget package in order to map properties automatically